internal/flags.go

```diff
@@ -144,6 +146,14 @@ func validateFlags(inputFlags *InputFlags) *ReturnError {
 		}
 	}
 
+	if inputFlags.checkBreaking && inputFlags.changelog {
+		return getErrInvalidFlags(fmt.Errorf("\"check-breaking\" and \"changelog\" cannot be used simultaneously"))
+	}
+
+	if len(inputFlags.includeChecks) > 0 && !inputFlags.checkBreaking {
```

Comment:
shouldn't this also be relevant to changelog?

Reply:
`changelog` subcommand will use all checkers.
Unlike `breaking` command that in case it finds breaking-changes, it will exit with an error code, the `changelog` will always return success (if the changelog can be generated of course). The clients can then filter out (based on the `id` and `level`) entries that are not considered relevant.
WDYT?

Otherwise, I can implement implement a ChangelogCheckConfig (similar to BackwardCompatibilityCheckConfig, but for now with all checkers) and allow the includeChecks option?